catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/SourcePoller.java

```diff
@@ -50,30 +50,27 @@
 
     /**
      * Constructor to schedule the SourcePollerRunner to execute immediately and at a fixed
-     * interval, currently set at every 60 seconds. This constructor is invoked by the
+     * period, currently set at every 60 seconds. This constructor is invoked by the
      * CatalogFramework's blueprint.
      *
-     * @param incomingRunner
-     *            the SourcePollerRunner to use for polling
+     * @param incomingRunner the SourcePollerRunner to use for polling
      */
     public SourcePoller(SourcePollerRunner incomingRunner) {
 
         this.runner = incomingRunner;
 
         scheduler = Executors.newSingleThreadScheduledExecutor();
 
-        handle = scheduler.scheduleAtFixedRate(runner, INITIAL_DELAY, INTERVAL, TimeUnit.SECONDS);
+        handle = scheduler.scheduleAtFixedRate(runner, initialDelay, period, TimeUnit.SECONDS);
```

Comment:
Now it will throw an exception when setting the interval something other than a positive, nonzero integer (and updated metatype description to reflect this).
done